ios/Integreat.xcodeproj/project.pbxproj

```diff
@@ -499,6 +495,30 @@
 					"$(PROJECT_DIR)/Pods/FirebaseAnalytics/Frameworks",
 					"$(PROJECT_DIR)/Pods/GoogleAppMeasurement/Frameworks",
 				);
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"$(inherited)",
+					"COCOAPODS=1",
+					"GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1",
+					"SD_WEBP=1",
+					"PB_FIELD_32BIT=1",
+					"PB_NO_PACKED_STRUCTS=1",
+					"PB_ENABLE_MALLOC=1",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_CLIENT_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_CLIENT_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_REVERSED_CLIENT_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_REVERSED_CLIENT_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_API_KEY=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_API_KEY)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_GCM_SENDER_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_GCM_SENDER_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_PLIST_VERSION=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_PLIST_VERSION)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_BUNDLE_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_BUNDLE_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_PROJECT_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_PROJECT_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_STORAGE_BUCKET=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_STORAGE_BUCKET)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_ADS_ENABLED=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_ADS_ENABLED)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_ANALYTICS_ENABLED=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_ANALYTICS_ENABLED)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_APPINVITE_ENABLED=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_APPINVITE_ENABLED)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_GCM_ENABLED=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_GCM_ENABLED)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_SIGNIN_ENABLED=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_IS_SIGNIN_ENABLED)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_GOOGLE_APP_ID=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_GOOGLE_APP_ID)\\\"",
+					"BUILD_CONFIG_IOS_GOOGLE_SERVICES_DATABASE_URL=\\\"$(BUILD_CONFIG_IOS_GOOGLE_SERVICES_DATABASE_URL)\\\"",
+				);
```

Comment:
What is this used for?

Reply:
This injects build config options as preprocessor macros